dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/test/configuration/MongoTestConfiguration.java

```diff
@@ -73,6 +73,11 @@ public MongoDbFactory mongoDbFactory(MongoClient mongoClient) throws Exception {
         return new SimpleMongoDbFactory(mongoClient, this.getDatabaseName());
     }
 
+    @Bean
+    public String mongoCollectionsFiles() {
```

Comment:
I don't see this used anywhere. Is this really needed?

Reply:
Yes, it is injected by the `@Document` annotation in the `VariantSourceMongo` class but not really used because we force the collection name by other means.